symfony/twig-bundle/5.4/templates/base.html.twig

```diff
@@ -14,6 +14,14 @@
         {% endblock %}
     </head>
     <body>
+        {% for type, messages in app.flashes %}
+            {% for message in messages %}
+                <div class="flash-{{ type }}">
```

Comment:
maybe add a general `flash` class, too, so all flash elements can be styled in a common way.

Reply:
Good call!